js/main.js

```diff
@@ -1,3 +1,7 @@
-import {map} from './map.js';
+import { getData } from './api.js';
+import {showMarkers} from './map.js';
 
-map;
+
+getData((offers) => {
+  showMarkers(offers);
+});
```

Comment:
```suggestion
getData(showMarkers);
```

ну и как то странная функция, что принимает и тут же сразу вызывает другую функцию. Может сразу showMakers и вызывать?